lib/sycamore/sycamore/utils/cache.py

```diff
@@ -39,10 +60,15 @@ def get_hash_context_file(file_path: str, hash_context: Optional[hashlib._Hash]
 
 class DiskCache(Cache):
     def __init__(self, cache_loc: str):
+        super().__init__()
         self._cache = diskcache.Cache(directory=cache_loc)
 
     def get(self, hash_key: str):
-        return self._cache.get(hash_key)
+        v = self._cache.get(hash_key)
+        if v:
```

Comment:
I'd use `is not None` here, just in case the hashed content is a zero-byte string.

Reply:
updated